Formula/xssp.rb

```diff
@@ -0,0 +1,35 @@
+class Xssp < Formula
+  desc "Create DSSP and HSSP files"
+  homepage "https://swift.cmbi.umcn.nl/gv/start/index.html"
+  url "https://github.com/cmbi/xssp/archive/3.0.5.tar.gz"
+  sha256 "fded09f08cfb12e578e4823295dc0d0aaeff6559d5e099df23c5bcc911597ccd"
+
+  depends_on "autoconf"
+  depends_on "automake"
+  depends_on "boost"
```

Comment:
`boost` may also be a `:build` dependency. Check the output of `brew linkage xssp`.

Reply:
The result of `brew linkage xssp` is
```
$ brew linkage xssp
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libc++.1.dylib
Homebrew libraries:
  /usr/local/opt/boost/lib/libboost_date_time-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_filesystem.dylib (boost)
  /usr/local/opt/boost/lib/libboost_iostreams-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_program_options-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_regex-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_system.dylib (boost)
  /usr/local/opt/boost/lib/libboost_thread-mt.dylib (boost)
  /usr/local/opt/boost/lib/libboost_timer.dylib (boost)
  /usr/local/opt/boost/lib/libboost_unit_test_framework-mt.dylib (boost)
```
Also, 
```
$ otool -L /usr/local/bin/mkdssp
/usr/local/bin/mkdssp:
        /usr/local/opt/boost/lib/libboost_date_time-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
        /usr/local/opt/boost/lib/libboost_filesystem.dylib (compatibility version 0.0.0, current version 0.0.0)
        /usr/local/opt/boost/lib/libboost_iostreams-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
        /usr/local/opt/boost/lib/libboost_program_options-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
        /usr/local/opt/boost/lib/libboost_regex-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
        /usr/local/opt/boost/lib/libboost_system.dylib (compatibility version 0.0.0, current version 0.0.0)
        /usr/local/opt/boost/lib/libboost_thread-mt.dylib (compatibility version 0.0.0, current version 0.0.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.200.5)
        /usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 400.9.4)
```
May I add a `:build` dependency to `boost`?

Reply:
No, there is run-time linkage to the `boost` libraries, so it is a run-time dependency. Some programs (like ABySS) only use the headers of `boost`, and have no run-time library linkage.